go/oasis-test-runner/scenario/e2e/identity_cli.go

```diff
@@ -54,7 +55,7 @@ func (i *identityCLIImpl) Run(childEnv *env.Env) error {
 		"identity", "init",
 		"--" + common.CfgDataDir, i.dataDir,
 	}
-	if err := runSubCommand(childEnv, "identity-init", i.nodeBinary, args); err != nil {
+	if err := cli.RunSubCommand(childEnv, i.logger, "identity-init", i.nodeBinary, args); err != nil {
```

Comment:
Not your fault but the receiver `i` makes me want to change it to `ident` or something.

Reply:
Yeah, good idea, done.